Modules/Loadable/Volumes/Logic/vtkSlicerVolumesLogic.cxx

```diff
@@ -1609,17 +1609,26 @@ void vtkSlicerVolumesLogic::InitializeDefaultVolumeDisplayPresets()
   std::string errorPrefix = "vtkSlicerVolumesLogic::InitializeDefaultVolumeDisplayPresets failed: Error reading '" + displayPresetsFilename + "'.";
 
   // Verify schema
-  if (!jsonRoot->HasMember("@schema"))
+  std::string schemaStr;
+  if (!jsonRoot->HasMember("$schema"))
```

Comment:
May be worth contributing the support for `$schema` in its own pull request. 

Some references related to the use of `@schema` vs `$schema` are also available at https://github.com/Slicer/Slicer/pull/6618#issuecomment-1292107500